### ISSUE TITLE
ALL Trade History 

### ISSUE THREAD

**Issue by xandyxor** [project-serum/serum-dex-ui#109]
How can see the ALL Trade History of pending orders and transaction record on dex?
NOT "Recent" Trade History

I think it can export or display all the trade history
Is a necessary function of a trading platform

**Comment by SmartDataco** [comment]
Agree,coming from a finance environment it is literally insane there appears to be no audit trail at all for historic transactions, after a few simple orders you are left with an explosion of rubbish data and no clear idea of what the heck the transactions were that got you to the current position, seriously need accountants onboard as much as coders if this ios going to work, right now its incomprehensible and therefore not usable and highly risky if you do use. Hope this is fixed properly with the aid of good accountants!

**Comment by gitmalong** [comment]
bump. can you at least outline the rpc logic how to retrieve filled orders for wallet address x for market y?

**Comment by fromzerotox** [comment]
As per my Research still no History available, how is that possible?

**Comment by gitmalong** [comment]
@armaniferrante any point regarding this?
